### PR TITLE
Build wheels on CI.

### PR DESCRIPTION
**Pull request by hameerabbasi**

### CODE REVIEW

**Comment by hameerabbasi** [comment]
Looks like we need `manylinux2`.

**Comment by hameerabbasi** [comment]
Looks like I was wrong, `manylinux1` has support for C++11.

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,6 @@
 include versioneer.py
 recursive-include uarray *.py
+include LICENSE
+recursive-include uarray *.cxx
```
Comment (rgommers) [inline MANIFEST.in]: this may leave something out. why not `recursive-include uarray *`? and then make sure to `prune` or `global-exclude` tempfiles

Comment (rgommers) [reply MANIFEST.in]: hmm at the moment there's only one file though

**Pull request merged**